requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy>=1.17
-pandas>=1.0.3,<1.1.0
+pandas>=1.0.3
```

Comment:
Should this be `>=1.1.0`? Tests currently pass on Pandas 1.0.5, but a lot of them are disabled...

Reply:
I was just about to test with lower versions. I think it would be better to have tests with at least 2 version of pandas, how about 1.0.5 and 1.1.0? Shall I setup travis to do this?